benchmarks/erp12/cbgp_lite/benchmark/suite/psb.clj

```diff
@@ -348,6 +783,34 @@
 
 (comment
 
-  (validate-solutions {:data-dir "data/psb/" :num-cases 10})
-
-)
\ No newline at end of file
+  (validate-solutions {:data-dir "data/psb/" :num-cases 50})
+
+  (read-cases {:data-dir "data/psb/"
+               :problem  "scrabble-score"
+               :n-test   10
+               :n-train  0})
+  ;; => {:train (),
+  ;;     :test
+  ;;     ({:inputs ["\n]BqAgxUre"], :output 27}
+  ;;      {:inputs ["ScG(5lh.%)tpK r9}"], :output 21}
+  ;;      {:inputs ["ul0vZvy\"'5;)6Q1"], :output 34}
+  ;;      {:inputs ["}6_"], :output 0}
+  ;;      {:inputs ["\tY\nh"], :output 8}
+  ;;      {:inputs ["D|NJ=n"], :output 12}
+  ;;      {:inputs ["@\n8}iDx(Q_uE.{"], :output 23}
+  ;;      {:inputs ["`R"], :output 1}
+  ;;      {:inputs ["i=yeVk\t`Jnw(;"], :output 28}
+  ;;      {:inputs ["{n2\t{\n@4W[\"]V4"], :output 9})}
+
+
+  (let [correct-string ""]
+    (map parse-long (str/split-lines correct-string)))
+
+  (str/split-lines "")
+  ;; => [""]
+
+  (str/split-lines "\n")
+  ;; => []
```

Comment:
I'm fine with leaving the `validate-solutions` line because most contributors will want to use that at the REPL as a test. The rest of these lines I think we should get rid of to avoid merge conflicts on everyones individual testing scratch code.

Reply:
Yup, good idea. I'll clean this up a bit and look into other instructions tomorrow.